### ISSUE TITLE
Data Import - Language missing or invalid

### ISSUE THREAD

**Issue by yan-ilumino** [mine-cetinkaya-rundel/accessible-cheatsheets#3]
**Page**: https://mine-cetinkaya-rundel.github.io/accessible-cheatsheets/data-import/data-import-accessible.html

**Issue**

The `<html lang>` attribute is missing or is empty, or a lang attribute value is not a valid language identifier.

Identifying the language of the page or page elements allows screen readers to read the content in the appropriate language. It also facilitates the automatic translation of content.

**How to fix it**

Identify the document language using the `<html lang>` attribute with a valid value (e.g., `<html lang="en">`). Ensure that all lang attribute values are valid.

**Success criteria**

- [3.1.1 Language of Page (Level A)](https://www.w3.org/WAI/WCAG21/quickref/#language-of-page1)

**Comment by yan-ilumino** [comment]
@mine-cetinkaya-rundel What you have now is sufficient: 

`<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">`
